downloaded/lisk_bridge.sh

```diff
@@ -21,45 +21,46 @@
 ######################################################################
 
 # Declare working variables
-TARGET_HEIGHT="3513100"
 BRIDGE_HOME="$(pwd)"
 BRIDGE_NETWORK="main"
-LISK_HOME="$HOME/lisk-main"
 
 # Reads in required variables if configured by the user.
 parseOption() {
 	OPTIND=1
-	while getopts ":s:b:n:h:" OPT; do
+	while getopts ":s:n:h:" OPT; do
 		# shellcheck disable=SC2220
 		case "$OPT" in
-			 s) LISK_HOME="$OPTARG" ;;
-			 b) BRIDGE_HOME="$OPTARG" ;; # Where the bridge is located
-			 n) BRIDGE_NETWORK="$OPTARG" ;; # Which network is being bridged
-			 h) TARGET_HEIGHT="$OPTARG" ;; # What height to cut over at
-		 esac
-	 done
+			s) LISK_HOME="$OPTARG" ;;
+			n) BRIDGE_NETWORK="$OPTARG" ;; # Which network is being bridged
+			h) TARGET_HEIGHT="$OPTARG" ;; # What height to cut over at
+			:) echo 'Missing argument for -'"$OPTARG" >&2;
+			   SHOW_USAGE=1;;
```

Comment:
Are we setting this variable to not duplicate code?
Not sure this is worth it (1 line here + 4 lines below vs. 2 lines)